Compiling with `default-features = false, , features = ["warp-compat"]` fails.

I use
```
dav-server = {version = "0.5.0", default-features = false, features = ["warp-compat"]
```

and get the following compile error:
```
   Compiling dav-server v0.5.7
error[E0432]: unresolved import `crate::localfs`
  --> /home/alex-dev/.cargo/registry/src/index.crates.io-6f17d22bba15001f/dav-server-0.5.7/src/warp.rs:11:29
   |
11 | use crate::{fakels::FakeLs, localfs::LocalFs, DavHandler};
   |                             ^^^^^^^ could not find `localfs` in the crate root

For more information about this error, try `rustc --explain E0432`.
error: could not compile `dav-server` (lib) due to previous error
```

Note that I didn't want to use the features `localfs` or `memfs` as I want to compile on targets without libc, e.g. Windows.

Maybe we should handle the feature in `warp.rs` correctly? If I find time, I'll prepare a PR.

--- a/src/warp.rs
+++ b/src/warp.rs
@@ -6,9 +6,12 @@
 //! rejection.
 //!
 use std::convert::Infallible;
+#[cfg(any(docsrs, feature = "localfs"))]
 use std::path::Path;
 
-use crate::{fakels::FakeLs, localfs::LocalFs, DavHandler};
+use crate::DavHandler;
+#[cfg(any(docsrs, feature = "localfs"))]
+use crate::{fakels::FakeLs, localfs::LocalFs};
 use warp::{filters::BoxedFilter, Filter, Reply};
 
 /// Reply-filter that runs a DavHandler.
@@ -83,6 +86,7 @@ pub fn dav_handler(handler: DavHandler) -> BoxedFilter<(impl Reply,)> {
 ///   the developers to this current limitation, so they don't accidentally expect
 ///   `auto_index_over_get` to control WebDAV.
 /// - no flags set: 404.
+#[cfg(any(docsrs, feature = "localfs"))]
 pub fn dav_dir(
     base: impl AsRef<Path>,
     index_html: bool,
@@ -105,6 +109,7 @@ pub fn dav_dir(
 
 /// Creates a Filter that serves a single file, ignoring the request path,
 /// like `warp::filters::fs::file`.
+#[cfg(any(docsrs, feature = "localfs"))]
 pub fn dav_file(file: impl AsRef<Path>) -> BoxedFilter<(impl Reply,)> {
     let handler = DavHandler::builder()
         .filesystem(LocalFs::new_file(file, false))

